Xamarin.Forms.Platform.GTK/Renderers/ImageRenderer.cs

```diff
@@ -207,4 +214,35 @@ public sealed class UriImageSourceHandler : IImageSourceHandler
 			return image;
 		}
 	}
+
+
+	public sealed class FontImageSourceHandler : IImageSourceHandler
+	{
+		public Task<Pixbuf> LoadImageAsync(ImageSource imageSource,
+			CancellationToken cancellationToken = new CancellationToken(), float scale = 1)
+		{
+			var fontImageSource = imageSource as FontImageSource;
+			var fontPathAndFamily = fontImageSource.FontFamily.Split('#');
+			var privateFontCollection = new PrivateFontCollection();
+			privateFontCollection.AddFontFile(fontPathAndFamily[0]);
+			var fontFamily = privateFontCollection.Families.FirstOrDefault(f => f.Name.Equals(fontPathAndFamily[1], StringComparison.InvariantCultureIgnoreCase));
+			var bmp = new Bitmap((int)fontImageSource.Size, (int)fontImageSource.Size);
+
+			using (var g = Graphics.FromImage(bmp))
+			{
+				var font = new DrawingFont(fontFamily, (int)fontImageSource.Size * .5f);
+				var fontColor = fontImageSource.Color != Xamarin.Forms.Color.Default ? fontImageSource.Color : Xamarin.Forms.Color.White;
+				g.DrawString(fontImageSource.Glyph, font, new SolidBrush(fontColor), 0, 0);
+			}
+
+			Pixbuf pixbuf;
+			using (var stream = new MemoryStream())
+			{
+				bmp.Save(stream, ImageFormat.Jpeg);
+				pixbuf = new Pixbuf(stream.GetBuffer());
```

Comment:
Is `GetBuffer` correct? It contains also unused bytes. Maybe use `stream.ToArray()` instead?
Also `bmp` can be disposed.